src/harness/harnessLanguageService.ts

```diff
@@ -526,6 +527,15 @@ module Harness.LanguageService {
         msg(message: string) {
             return this.host.log(message);
         }
+        
+        enabled() {

```

Comment:
Enabled is a very generic term.  I can't really tell what functionality this is supposed to represent.